django_freeradius/api/views.py

```diff
@@ -28,7 +28,10 @@ def authenticate(self, request):
         if request.META.get('HTTP_AUTHORIZATION', False):
             headers = request.META.get('HTTP_AUTHORIZATION').split(',')
             for header in headers:
-                token = header.split(' ')[1]
+                try:
+                    token = header.split(' ')[1]
+                except IndexError:
+                    token = header[6:]
```

Comment:
It's not a good practice to hide an error of the client. In this case the client is sending bogus data and we need to let them know.

Do the following:

```
raise ParseError('Invalid token')
```

You will need to import `ParseError`, add it to the end of line 10.

Reply:
okay, so instead of fixing the invalid token it should raise a parse error of invalid token?